### ISSUE TITLE
New terms request under: Activation function

### ISSUE THREAD

**Issue by hrshdhgd** [berkeleybop/artificial-intelligence-ontology#12]
For new term requests, please provide the following information:

## Preferred term labels
 - deserialize [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/deserialize)
 - elu [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/elu)
 - exponential [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/exponential)
 - gelu [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/gelu)
 - hard_sigmoid [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/hard_sigmoid)
 - linear [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/linear)
 - relu [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/relu)
 - selu [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/selu)
 - serialize [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/serialize)
 - sigmoid [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/sigmoid)
 - softmax [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/softmax)
 - softplus [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/softplus)
 - softsign [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/softsign)
 - swish [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/swish)
 - tanh [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/activations/tanh)

## Synonyms

Not aware of any. 

## Textual definition

the definition should be in the linked docs above. If not we can decide on a standard definition.

## Suggested parent term

Please look in the hierarchy in a browser such as [OLS](http://www.ebi.ac.uk/ols/ontologies/deep-learning-ontology)
All of the come under `Activation function`

## Attribution

If you would like a nanoattribution, please indicate your ORCID id

**Comment by hrshdhgd** [comment]
`Layers` could also be another tree that can be added [tensorflow link](https://www.tensorflow.org/api_docs/python/tf/keras/layers)
